packages/paste-codemods/tools/.cache/mappings.json

```diff
@@ -18,6 +18,7 @@
   "BreadcrumbItem": "@twilio-paste/core/breadcrumb",
   "ButtonGroup": "@twilio-paste/core/button-group",
   "Button": "@twilio-paste/core/button",
+  "ToggleButtonStyles": "@twilio-paste/core/button",
```

Comment:
For the reason in which we create a mapping from package -> component exports, we have this loosely defined rule that exports from any package should be prefixed with the package name.

Can you instead export the styles as something like `ButtonToggleStyles` so there's no theoretical clash in the future and the mappings stay "obvious" please?